src/com/facebook/buck/android/TrimUberRDotJava.java

```diff
@@ -212,9 +212,12 @@ public Pattern apply(@NonNull String input) {
       // is referenced.  That is a very rare case, though, and not worth the complexity to fix.
       if (m.find()) {
        final String resource = m.group(1);
-       if (!allReferencedResources.contains(packageName + "." + resource) &&
-           (pattern.isPresent() && !pattern.get().matcher(resource).find())) {
-             continue;
+       if (!allReferencedResources.contains(packageName + "." + resource)) {
+         if (pattern.isPresent() && pattern.get().matcher(resource).find()) {
```

Comment:
It was before, but that caused the integration test to fail. This is because the continue needs to happen for both conditionals, but the write only needs to happen if the inner conditional is satisfied